packages/twenty-website/src/content/user-guide/functions/workflows.mdx

```diff
@@ -0,0 +1,151 @@
+---
+title: Workflows
+info: Understand the workflow feature in Twenty including how to create and delete workflows, configure triggers, add actions, run workflows, and manage workflow history.
+image: /images/user-guide/workflows/workflow.png
+sectionInfo: Learn how to connect Twenty to your other tools.
+---
+
+## Create a Workflow
+
+To create a workflow:
+
+1. Click “+ Add a Workflow” to begin
+2. Click “Untitled” and enter your workflow name on the upper left 
+3. Once you have created one workflow, you can click “+ New record” to create additional workflows
+
+## Delete a Workflow
+
+To delete a workflow, click **Delete** in the top right corner of the workflow record. Confirm the deletion in the pop-up modal.
+
+To view deleted workflows:
+
+- Use the **Filter** option, or
+- Open quick action side panel using `⌘K` and select **See deleted workflows**.
+
+You can also save this view by clicking **Save as new view** for quick access later.
+
+## Configure a Trigger
+
+Workflows always start with a single trigger. It is the condition that activates your workflow. In this step, you decide when and how the automation should start. There are several types of triggers available. In this section, you’ll learn about each trigger type and how to configure one.
+
+### Record is Created 
+
+This trigger starts the workflow when a record is created in a selected object (e.g. People, Companies, or any custom object). To configure this trigger select the record type where the workflow should listen for a new entry.
+
+### Record is Updated
+
+This trigger starts the workflow when changes are made to an existing record. To configure this trigger, select the record type and optionally select the field(s) you want to listen for changes.
+
+### Record is Deleted
+
+This trigger starts the workflow when a record is removed. To configure this trigger, select the record type you want to listen for deletions.
+
+### Launch Manually
+
+This trigger starts the workflow only when manually evoked by you. You can trigger it in two ways:
+
+- When record is selected lets you use data from that record in later steps; you'll need to choose the object, or 
+- When no record is selected runs the workflow without any record context.
+
+To manually trigger the workflow, open the side panel with `⌘K` and select the workflow. Once the run completes, reload the page to see the changes.
```

Comment:
I think we could precise how the user can launch such a workflow.

1. On the record table, select one record avec open the side panel. You should be able to launch the workflow with the selected record.
2. If you created a workflow with a manual trigger without involving record selection, you can trigger it from any page on the app.

Something like this 🙂